test/ui-test/testSuites/suite_messaging/tst_chatsNavigation/test.py

```diff
@@ -0,0 +1,9 @@
+source(findFile('scripts', 'python/bdd.py'))
+
+setupHooks('bdd_hooks.py')
+collectStepDefinitions('./steps', '../shared/steps/', '../../global_shared/steps/', '../../suite_onboarding/shared/steps/')
```

Comment:
I think you can remove the dependence to `suite_onboarding` here bc you are not using any step from there in your feature file. You are adding the onboarding steps into the `hooks` file.